pull-request.sh

```diff
@@ -105,12 +106,16 @@ create_pull_request() {
                 DATA="{\"assignees\":[${ASSIGNEES}]}"
                 echo "${DATA}"
                 ASSIGNEES_URL="${ISSUE_URL}/${NUMBER}/assignees"
-                curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X POST --data "${DATA}" ${ASSIGNEES_URL}
-                RETVAL=$?
+                RETVAL=0
+                if ! RESPONSE="$(safe_curl -X POST --data "${DATA}" ${ASSIGNEES_URL})"; then
```

Comment:
We should be setting/printing the return values (so the user can see them, for example) before this is run, otherwise the function will exit without any meaningful error message.

Reply:
centralizing conversation in https://github.com/vsoch/pull-request-action/pull/39#discussion_r467147007